src/Auth.ts

```diff
@@ -485,6 +496,12 @@ export class Auth {
     });
   }
 
+  private calculateThumbprint(certificate: pki.Certificate) {
```

Comment:
Let's specify return type to avoid refactoring issues in the future

Reply:
Done.